net/sshtunnel/files/sshtunnel.init

```diff
@@ -178,6 +179,17 @@ load_server() {
 		LogLevel PKCS11Provider ServerAliveCountMax ServerAliveInterval \
 		StrictHostKeyChecking TCPKeepAlive VerifyHostKeyDNS
 
+	if [ -n "$ProxyCommand" ]; then
+		ProxyCommand=$(echo $ProxyCommand | sed "s/%h/"${hostname}"/g")
```

Comment:
FYI: You can make multiple replacements in a single sed command:

`sed -e 's/the/hello/g' -e 's/lazy/tired/' test.txt`

So here it can be used something like:
`ProxyCommand=$(echo $ProxyCommand | sed -e "s/%h/"${hostname}"/g" -e "s/%p/"${port}"/g")`

But I didn't tested.
I think here we can use a built-in shell replacement  
https://stackoverflow.com/a/3306054/1049542